src/applications/personalization/profile360/components/PhoneEditModal.jsx

```diff
@@ -2,9 +2,27 @@ import React from 'react';
 import { merge } from 'lodash';
 
 import ErrorableTextInput from '@department-of-veterans-affairs/formation/ErrorableTextInput';
+import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
 
 import Vet360EditModal from './Vet360EditModal';
 
+class PhoneTextInput extends ErrorableTextInput {
+  componentDidMount() {
```

Comment:
this is obtuse but unfortunately the most straightforward way to make this styling change to the inputs without copying the ErrorableTextInput component out of the design system library

Reply:
Yeah, a good example of how we need more flexibility in our design system's form components. They could use a lot of sprucing up IMO. There should also be a more graceful way to apply validation.